wand-watcher.lic

```diff
@@ -0,0 +1,153 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#wand-watcher
+=end
+
+no_kill_all
+no_pause_all
+
+custom_require.call(%w[common common-items])
+
+class WandWatcher
+  include DRC
+  include DRCI
+
+  def initialize
+    settings = get_settings();
+    @wand_list = settings.wands
+    @queued_wands = []
+    @no_use_scripts = settings.wand_watcher_no_use_scripts || []
```

Comment:
Proposing list of rooms where not to try and use the wands, such as banks and vaults or other areas where magical devices are restricted.

```suggestion
    @no_use_scripts = settings.wand_watcher_no_use_scripts || []
    @no_use_rooms = settings.wand_watcher_no_use_rooms || []
```